x-pack/plugins/index_management/server/routes/api/data_streams/register_get_route.ts

```diff
@@ -79,8 +79,8 @@ const getDataStreams = (client: ElasticsearchClient, name = '*') => {
   });
 };
 
-const getDataStreamsStats = (client: ElasticsearchClient, name = '*') => {
-  return client.transport.request({
+const getDataStreamsStats = (client: IScopedClusterClient, name = '*') => {
+  return client.asCurrentUser.transport.request({
```

Comment:
Does this comment apply here as well, or can we use `client.indices.dataStreamsStats`?
```
  // TODO update when elasticsearch client has update requestParams...
```

Reply:
Good catch, fixed.